server/conf/evolutions/default/66.sql

```diff
@@ -0,0 +1,66 @@
+# --- !Ups
+--Drop the existing view
+DROP MATERIALIZED VIEW IF EXISTS monthly_submissions_reporting_view;
+-- create a new view
+CREATE MATERIALIZED VIEW IF NOT EXISTS monthly_submissions_reporting_view AS
+  SELECT
+  programs.name AS program_name,
+  activeprogram.localized_name AS localized_name,
+  date_trunc('month', applications.submit_time) AS submit_month,
+  count(*),
+  percentile_cont(0.25) WITHIN GROUP (
+  ORDER BY applications.submission_duration) AS p25,
+  percentile_cont(0.5) WITHIN GROUP (
+  ORDER BY applications.submission_duration) AS p50,
+  percentile_cont(0.75) WITHIN GROUP (
+  ORDER BY applications.submission_duration) AS p75,
+  percentile_cont(0.99) WITHIN GROUP (
+  ORDER BY applications.submission_duration) AS p99
+  FROM applications
+  INNER JOIN programs ON applications.program_id = programs.id
+  INNER JOIN
+  (SELECT
+    p.name,
+    ((p.localized_name #>> '{}')::jsonb #>> '{translations,en_US}') AS localized_name
```

Comment:
praise: `'{translations,en_US}'`

Thanks for teaching me something new!

Reply:
@dkatzz figured it out!